packages/util/src/swap/types.ts

```diff
@@ -20,6 +20,10 @@ export interface ReceiverHistoryItem {
   assetId: string;
 }
 
+export interface SwapTransferBatchAdditionalData {
+  rates: Array<string>;
```

Comment:
would be nice to know what string is

type what = string

rates: Array<what>

Reply:
SONAR can complain though